### PR TITLE
ARM64 via CCI machine

### LINKED ISSUE #1007: Change CircleCI to use native arm64 runners
https://circleci.com/docs/2.0/arm-resources/

### PR DESCRIPTION
**Pull request by baentsch**
Fixes #1007 

* [ ] Does this PR change the input/output behaviour of a cryptographic algorithm (i.e., does it change known answer test values)?  (If so, a version bump will be required from *x.y.z* to *x.(y+1).0*.)
* [ ] Does this PR change the the list of algorithms available -- either adding, removing, or renaming?  (If so, PRs in OQS-OpenSSL, OQS-BoringSSL, and OQS-OpenSSH will also be required by the time this is merged.)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -346,10 +375,16 @@ workflows:
...
+      #- arm_emulated:
+      #    <<: *require_buildcheck
+      #    name: arm64
+      #    ARCH: arm64
+      #    CMAKE_ARGS: -DOQS_ENABLE_SIG_SPHINCS=OFF -DOQS_DIST_BUILD=ON
+      - arm_machine:
           <<: *require_buildcheck
           name: arm64
-          ARCH: arm64
+          PYTEST_ARGS: --ignore=tests/test_portability.py --numprocesses=auto --maxprocesses=10
```
Comment (jschanck) [inline .circleci/config.yml:387]: We should run test_portability. We don't have any code dispatching on ARM yet, but we will soon.

File: .circleci/config.yml [hunk 2]
```diff
@@ -346,10 +375,16 @@ workflows:
...
           CMAKE_ARGS: -DOQS_ENABLE_SIG_SPHINCS=OFF -DOQS_DIST_BUILD=ON
```
Comment (jschanck) [inline .circleci/config.yml:388]: The test is completing in <4 minutes, so let's run SPHINCS.

**Review by jschanck — APPROVED** [review]
Looks good. Two minor suggestions above. Both are optional.

File: .circleci/config.yml (hunk 1 above)
Comment (baentsch) [reply .circleci/config.yml]: Done with update

File: .circleci/config.yml (hunk 2 above)
Comment (baentsch) [reply .circleci/config.yml]: Done with update

**Pull request merged**